contracts/modules/cover/Cover.sol

```diff
@@ -240,12 +235,20 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon {
         requestedCoverAmountInNXM
       );
 
+      // apply the global rewards ratio and the total Rewards in NXM
+      uint totalRewardsInNXM = premiumInNXM * globalRewardsRatio / REWARD_DENOMINATOR;
```

Comment:
I'm doing the same calculation in the staking contract. Let's return it from the staking contract so we'd have it calculated in a single place, less room for error.